<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startCleavizButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Cleaviz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startMockButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Mock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startGrinderButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="text">
    <string>Start Grinder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopMockButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Mock</string>
   </property>
  </widget>
  <widget class="QWidget" name="runningStatusBar" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="stopGrinderButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Grinder</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>451</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="bucketingButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Bucketing</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="plottingButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Plotting</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="peakDetectionButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Peak Detection</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="peakDetectionSummaryButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Peak Detection Summary</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="simpleMovingAverageButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Simple Moving Average</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="ridgeRegressionButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ridge Regression</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="spectralAnalysisButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Spectral Analysis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Examples</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="stimuliSetupButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>SETUP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="stimuliStartButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="stimuliFlashButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>FLASH</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="stimuliStopButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Stimuli</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
